libs/store/src/lib/infrastructure/persistence/store/entity-store.spec.ts

```diff
@@ -2,18 +2,23 @@ import { of } from 'rxjs';
 
 import { DefaultEntityStore } from './entity-store';
 import { QueryBuilder } from '../query/query-builder';
-import { Type } from '../../../domain/utility'
+import { Type } from '../../../domain/utility';
 import { IdentityKey } from '../../../domain/entity';
 import { BaseEntity } from '../domain/base-classes/base-entity';
 import { EntityCollectionService } from './entity-collection-service';
 
-class User extends BaseEntity<{}>{
-    constructor(public id: IdentityKey, public name: string, public age: number) {
-        super({ id, name, age });
+class UserDTO {
+    id: IdentityKey;
+    name: string;
+    age: number;
+}
+class User extends BaseEntity<UserDTO> {
+    constructor(dto: UserDTO) {
```

Comment:
These examples are not aligned, somewhere we are constructor and accessing .value and somewhere not.